.github/workflows/jekyll-build.yml

```diff
@@ -0,0 +1,16 @@
+name: Jekyll Build Verification
+
+on: [pull_request]
```

Comment:
Maybe we should also run an all/external link checker on a cron schedule (every week?) because sometimes external links change over time. We can create an issue if it fails. Also, does this run a link checker on all files on every PR? If we add a run on a schedule, then I think it may be sufficient to just check changed files on every PR.